index.ts

```diff
@@ -21,6 +32,14 @@ import { dedent } from 'ts-dedent'
 
 let local: Awaited<ReturnType<typeof launch>>
 
+type ShardItem = {
+  ShardIterator: string
+}
+
+let continueProcessingLoop = true
```

Comment:
You need to set `continueProcessingLoop` to `true` in your `start` entry point, or else the table streams invocations will cease permanently after Architect has restarted the sandbox.